packages/bot-web-ui/src/components/bot-snackbar/bot-snackbar.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Icon, Toast } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TBotSnackbar = {
+    className?: string;
+    is_open: boolean;
+    onClick?: React.MouseEventHandler<HTMLDivElement>;
+    handleClose: () => void;
+    type?: 'error' | 'info' | 'notification';
+    timeout?: number;
+    msg_localize_components?: JSX.Element[];
+    message: string;
+};
+
+const BotSnackbar = ({
+    message,
+    msg_localize_components = [],
+    timeout = 6000,
+    is_open,
+    onClick,
+    handleClose,
+    type,
+    className,
+}: TBotSnackbar) => {
+    const [notification_timer, setNotificationTimer] = useState(timeout);
+
+    return (
+        <div
+            onMouseOver={() => {
+                setNotificationTimer(20000);
```

Comment:
Am I reading this right, that when you hover the notification will stick around for 20 seconds (instead of 6) and then still disappear?

Would it be better if we never hide the notification if they are hovering over it?

Reply:
Yes, @markwylde-deriv! It would be ideal not to close it if the user hovers over it. But the `Toast` component from the `@deriv/components` uses this timeout to show/hide the snack bar. That's why I was putting in a maximum of 20 seconds delay before closing this.

Reply:
@markwylde-deriv I have checked the `Toast` component and seems like if we pass 0 for the timeout then it doesn't run the `setTimeout`. So, this should solve the issue. Now on mouse over the snack bar should keep the snack bar open until the mouse leaves.

Reply:
Awesome! Nice one!